tests/backup/backup_helper.go

```diff
@@ -3222,6 +3222,35 @@ func ValidatePodByLabel(label map[string]string, namespace string, timeout time.
 	return nil
 }
 
+// IsMongoDBReady validates if the mongo db pods in Px-Backup namespace are healthy enough for Px-Backup to function
+func IsMongoDBReady() error {
+	log.Infof("Verify that at least 2 mongodb pods are in Ready state at the end of the testcase")
+	pxbNamespace, err := backup.GetPxBackupNamespace()
+	if err != nil {
+		return err
+	}
+	mongoDBPodStatus := func() (interface{}, bool, error) {
+		statefulSet, err := apps.Instance().GetStatefulSet(mongodbStatefulset, pxbNamespace)
+		if err != nil {
+			return "", true, err
+		}
+		// Px-Backup would function with just 2 mongo DB pods in healthy state.
+		// Ideally we would expect all 3 pods to be ready but because of intermittent issues, we are limiting to 2
+		if statefulSet.Status.ReadyReplicas >= 2 {
```

Comment:
Is it not statefulSet.Status.ReadyReplicas < 2?

Reply:
Fixed it